release/README.md

```diff
@@ -100,13 +100,10 @@ Working in your fork of `google-cloud-cpp`: bump the version numbers to the
 send the PR for review against `main`. You need to:
 
 - Update the version number in the top-level `CMakeLists.txt` file.
-- Run the cmake configuration step, which will update the
-  `google/cloud/internal/version_info.h` file. Hint: Running
-  `ci/cloudbuild/build.sh -t clang-tidy-pr` will do this, and you can `CTRL-C`
-  the build after CMake's configure step is finished.
 - Update the ABI baseline to include the new version numbers in the inline
   namespace by running `ci/cloudbuild/build.sh -t check-api-pr`, which will
   leave the updated files in `ci/abi-dumps`.
+- This will also update the `google/cloud/internal/version_info.h` file.
```

Comment:
nit: I'd append this to the previous bullet, because it's not an action to take.